src/app/components/select/select.js

```diff
@@ -5,6 +5,8 @@ import {description as template} from './select.html';
 const CONVERT_OPTIONS = Symbol();
 const CLOSE_DROPDOWNS = Symbol();
 const CONTROLLERS = Symbol();
+const STOP_SCROLLING = Symbol();
```

Comment:
Correct. And they actually aren't truly private (it's possible to access them with `Object.getOwnPropertySymbols`). I try to put it on components that may be inherited/overloaded though to avoid name collisions and make it clear what should be considered an internal method without the `_` convention.